pkg/configuration/configuration.go

```diff
@@ -250,3 +260,7 @@ func (c *Config) GetDeactivationDomainsExcludedList() []string {
 		return c == ','
 	})
 }
+
+func (c *Config) GetForbiddenUsernamePrefixes() []string {
+	return c.host.GetStringSlice(varForbiddenUsernamePrefixes)
```

Comment:
Since you didn't add a test for the case when it would be set using a variable, could you please clarify what string format is expected in this case? I tried comma-separated string, but that doesn't work.